frontend/src/components/invoice/invoice-edit/EditInvoice.tsx

```diff
@@ -51,11 +51,12 @@ const EditInvoice = () => {
     setInvoice(new InvoiceModel(config, storeInvoice));
   }
 
+  const configRef = useRef(config)
   useEffect(() => {
     const isQuotation = window.location.pathname.startsWith('/quotations/');
     const navigateInvoice = invoices.filter(x => x.isQuotation && isQuotation).find(x => x.number === parseInt(params.id, 10))
-    setInvoice(new InvoiceModel(config, navigateInvoice))
-  }, [params, config, invoices])
+    setInvoice(new InvoiceModel(configRef.current, navigateInvoice))
+  }, [params, invoices])
```

Comment:
Hehehehehehe....

Is dit een gevolg van mijn comment?
Want je kan dan nu exact hetzelfde doen:
- Tab 1: Open invoice, maak wijziging, save niet
- Tab 2: editeer een andere invoice & save